src/set_theory/game/basic.lean

```diff
@@ -75,23 +75,35 @@ quotient.lift₂ lf (λ x₁ y₁ x₂ y₂ hx hy, propext (lf_congr hx hy))
 
 local infix ` ⧏ `:50 := lf
 
-/-- On `pgame`, simp-normal inequalities should use as few negations as possible. -/
+/-- On `game`, simp-normal inequalities should use as few negations as possible. -/
 @[simp] theorem not_le : ∀ {x y : game}, ¬ x ≤ y ↔ y ⧏ x :=
 by { rintro ⟨x⟩ ⟨y⟩, exact pgame.not_le }
 
-/-- On `pgame`, simp-normal inequalities should use as few negations as possible. -/
+/-- On `game`, simp-normal inequalities should use as few negations as possible. -/
 @[simp] theorem not_lf : ∀ {x y : game}, ¬ x ⧏ y ↔ y ≤ x :=
 by { rintro ⟨x⟩ ⟨y⟩, exact pgame.not_lf }
 
 instance : is_trichotomous game (⧏) :=
 ⟨by { rintro ⟨x⟩ ⟨y⟩, change _ ∨ ⟦x⟧ = ⟦y⟧ ∨ _, rw quotient.eq, apply lf_or_equiv_or_gf }⟩
 
+/-! The `add_comm_group` structure on `game` often simplifies many proofs of inequalities on
+`pgame`. -/
+
```

Comment:
This comment feels weird because none of the examples below are using the additive structure.

Reply:
I'm trying to explain why you'd want to cast inequalities from `pgame` to `game` to begin with.

Reply:
Then perhaps

> It can be useful to use these lemmas to turn `pgame` inequalities into `game` inequalities, as the `add_comm_group` structure on `game` often simplifies many proofs

Your current comment doesn't make any explicit reference to the lemmas.